src/components/SQForm/useForm.tsx

```diff
@@ -78,7 +78,9 @@ export function useForm<TValue, TChangeEvent>({
   const isDirty = !isEqual(meta.initialValue, meta.value);
   const hasValue = _getHasValue(meta);
   const isError = !!errorMessage;
-  const isRequired = errorMessage?.toLowerCase() === 'required';
+  const isRequired = Array.isArray(errorMessage)
```

Comment:
the loom shows .toString() but I actually switched to .join("") so that it cuts out the commas and returns us a regular string that is not comma-separated.